src/solver/optimisation/post_process_commands.cpp

```diff
@@ -140,25 +139,34 @@ void DTGmarginForAdqPatchPostProcessCmd::execute(const optRuntimeData&)
         if (problemeHebdo_->adequacyPatchRuntimeData->areaMode[Area] != physicalAreaInsideAdqPatch)
             continue;
 
+        auto& hourlyResults = problemeHebdo_->ResultatsHoraires[Area];
+        const auto& scratchpad = area_list_[Area]->scratchpad[thread_number_];
+        const double unsuppliedEnergyCost = area_list_[Area]->thermal.unsuppliedEnergyCost;
+
         for (uint hour = 0; hour < nbHoursInWeek; hour++)
         {
-            auto& hourlyResults = problemeHebdo_->ResultatsHoraires[Area];
-            const auto& scratchpad = area_list_[Area]->scratchpad[thread_number_];
+            const bool isHourTriggeredByCsr = problemeHebdo_->adequacyPatchRuntimeData
+                    ->wasCSRTriggeredAtAreaHour(Area, hour);
+
             const double dtgMrg = scratchpad.dispatchableGenerationMargin[hour];
             const double ens = hourlyResults.ValeursHorairesDeDefaillancePositive[hour];
-            const bool triggered = problemeHebdo_->adequacyPatchRuntimeData
-                                     ->wasCSRTriggeredAtAreaHour(Area, hour);
-            hourlyResults.ValeursHorairesDtgMrgCsr[hour] = recomputeDTG_MRG(triggered, dtgMrg, ens);
-            hourlyResults.ValeursHorairesDeDefaillancePositiveCSR[hour] = recomputeENS_MRG(
-              triggered,
-              dtgMrg,
-              ens);
-
-            const double unsuppliedEnergyCost = area_list_[Area]->thermal.unsuppliedEnergyCost;
-            hourlyResults.CoutsMarginauxHoraires[hour] = recomputeMRGPrice(
-              hourlyResults.ValeursHorairesDtgMrgCsr[hour],
-              hourlyResults.CoutsMarginauxHoraires[hour],
-              unsuppliedEnergyCost);
+
```

Comment:
Previous calls to functions : 
- **recomputeDTG_MRG**
- **recomputeENS_MRG**
- **recomputeMRGPrice**

were removed, and replaced with plain instructions here (in the caller).
Indeed, these functions were judged overkill (having function for the very little thing they do is overkill), unclear (they hide the use of the **triggered** parameter, which is important in the caller), and with too many arguments for their very little responsibility.